recipes/python-gilt/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "python-gilt" %}
+{% set version = "1.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: e23a45a6905e6bb7aec3ff7652b48309933a6991fad4546d9e793ac7e0513f8a
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  entry_points:
+    - gilt = gilt.shell:main
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools
+    - pbr
+  run:
+    - pbr
+    - python
+
+test:
+  imports:
+    - gilt
+
+about:
+  home: https://github.com/metacloud/gilt
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: 'gilt - A GIT layering tool.'
+  description: |
+    Install gilt using pip:
+    .. code-block:: bash
+
+        $ pip install python-gilt
+
+    Overlay remote repositories and/or files into the destinations provided.
+
+    .. code-block:: bash
+
+        $ gilt --debug overlay
```

Comment:
I think that is not a good description, could you please fix it?

Reply:
Done